README.md

```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
 
 That's all you need to do for syncing within the editor to work. To perform interactive queries against the graph, once the graph is Active, click 'Manage' and then on the next page, click 'Open Browser'. If you're unfamiliar with the browser or Neo4j in general, check out the [docs](https://neo4j.com/developer/neo4j-browser/).
 
-Optional Steps:
-To link data to ESCO you will need to install the ESCO TTL file into Neo4J, there are a number of steps that need to be followed and executed, for more information on how to do this view [Neo4J Setup](https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1491501074/SPIKE+RDF+hosting+-+Neo4j+cluster) on Confluence. (This link is only available to Internal users)
+#### Loading ESCO data
+
+1) Install the APOC plugin - go to plugins -> install
+2) Install the Neosemantics v4 plugin - https://github.com/neo4j-labs/neosemantics - copy jar to plugins directory, add to the end of neo4j.conf - dbms.unmanaged_extension_classes=n10s.endpoint=/rdf
+    NOTE: If any existing plugins have already set "dbms.unmanaged_extension_classes", combine the existing values with the required Neosemantics value, by comma separating them. (E.g. if both the Neosemantics and GraphQL plugins are installed, set the value to dbms.unmanaged_extension_classes=org.neo4j.graphql=/n10s.endpoint=/rdf)
+3) Restart graph instance
+4) Enable the browser setting "Enable multi statement query editor"
+5) NOTE: Use http://neo4j-labs.github.io/neosemantics/ for reference:
+    Using the desktop app - query window:
+    CREATE CONSTRAINT ON (n:Resource) ASSERT n.uri IS UNIQUE;
+    Add the namespace prefixes by calling :
+    call n10s.nsprefixes.add("adms","http://www.w3.org/ns/adms#");
```

Comment:
We don't need all these, and we need to create one for regulated-professions. Take a look [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/Neo4jScripts/neo4j-db-config).